### PR TITLE
User-Guide.iypnb: Link IO-notebook, correct degree distribution figure.

### PR DESCRIPTION
**Pull request by fabratu**

### CODE REVIEW

File: notebooks/User-Guide.ipynb [hunk 1]
```diff
@@ -1131,6 +1140,13 @@
    "source": [
     "NetworKit is an open-source project that improves with suggestions and contributions from its users. The [mailing list](https://sympa.cms.hu-berlin.de/sympa/subscribe/networkit) is the place for general discussion and questions."
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
Comment (angriman) [inline notebooks/User-Guide.ipynb:1149]: Is this an empty cell?

File: notebooks/User-Guide.ipynb [hunk 2]
```diff
@@ -1149,9 +1165,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "version": "3.7.3"
```
Comment (angriman) [inline notebooks/User-Guide.ipynb:1168]: This is probably unwanted.

File: notebooks/User-Guide.ipynb [hunk 3]
```diff
@@ -1149,9 +1165,9 @@
...
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 2
```
Comment (angriman) [inline notebooks/User-Guide.ipynb:1172]: Same here.

**Comment by fabratu** [comment]
Yeah, right. I don't think the versioning references make any difference - however I reverted them (again). Seems I have to be more careful to not let a notebook server open while commiting notebook changes.

**Review by angriman — APPROVED** [review]

**Pull request merged**